src/libmongoc/doc/mongoc_client_encryption_opts_set_key_expiration.rst

```diff
@@ -0,0 +1,27 @@
+:man_page: mongoc_client_encryption_opts_set_key_expiration
```

Comment:
Add `mongoc_client_encryption_opts_set_key_expiration` to the `toctree` of `mongoc_client_encryption_opts_t.rst` to include in the docs.